commit 220bba7fe952d4aed2cc2dd83748e6b607c8253d
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Tue Jan 16 12:59:17 2018 +0000

    overlays: Fix resetgpio and ledgpio for hy28a/b
    
    Offsets for overlay parameters are specified in bytes, not in access
    units.
    
    See: https://github.com/raspberrypi/linux/issues/2344
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/boot/dts/overlays/hy28a-overlay.dts b/arch/arm/boot/dts/overlays/hy28a-overlay.dts
index ac0f3c28ea85..d7625a6372f5 100644
--- a/arch/arm/boot/dts/overlays/hy28a-overlay.dts
+++ b/arch/arm/boot/dts/overlays/hy28a-overlay.dts
@@ -86,8 +86,8 @@
 		debug =		<&hy28a>,"debug:0";
 		xohms =		<&hy28a_ts>,"ti,x-plate-ohms;0";
 		resetgpio =	<&hy28a>,"reset-gpios:4",
-				<&hy28a_pins>, "brcm,pins:1";
+				<&hy28a_pins>, "brcm,pins:4";
 		ledgpio =	<&hy28a>,"led-gpios:4",
-				<&hy28a_pins>, "brcm,pins:2";
+				<&hy28a_pins>, "brcm,pins:8";
 	};
 };
diff --git a/arch/arm/boot/dts/overlays/hy28b-overlay.dts b/arch/arm/boot/dts/overlays/hy28b-overlay.dts
index 8018aebde8e5..70c1118064b6 100644
--- a/arch/arm/boot/dts/overlays/hy28b-overlay.dts
+++ b/arch/arm/boot/dts/overlays/hy28b-overlay.dts
@@ -141,8 +141,8 @@
 		debug = 	<&hy28b>,"debug:0";
 		xohms =		<&hy28b_ts>,"ti,x-plate-ohms;0";
 		resetgpio =	<&hy28b>,"reset-gpios:4",
-				<&hy28b_pins>, "brcm,pins:1";
+				<&hy28b_pins>, "brcm,pins:4";
 		ledgpio =	<&hy28b>,"led-gpios:4",
-				<&hy28b_pins>, "brcm,pins:2";
+				<&hy28b_pins>, "brcm,pins:8";
 	};
 };
